tools/pot/tests/test_scales.py

```diff
@@ -16,6 +16,9 @@
 from openvino.tools.pot.graph import model_utils as mu
 from openvino.tools.pot.statistics.collector import StatisticsCollector
 
+
+EPS = 1e-6
```

Comment:
Why did you set this to be stronger condition (`1e-3` to `1e-6`)?

Reply:
As we previously discussing with @AlexKoff88, we are decreasing eps to minimize bug possibility. We had margin "just in case" before, and we decided to shrink this margin